index.html

```diff
@@ -0,0 +1,161 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <script
+      src="https://kit.fontawesome.com/43fc886676.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Gaming Convention</title>
+  </head>
+  <body>
+    <div id="container">
+      <header class="head-section">
+        <div class="hamburger-icon">
+          <button class="hamburger-button" type="button">
+            <span class="h-button"></span>
+            <span class="h-button"></span>
+            <span class="h-button"></span>
+          </button>
+        </div>
+        <div class="nav-bars">
+          <nav class="nav-bar-one">
+            <ul>
+              <li>
+                <a href="#"><i class="fa-brands fa-facebook-f"></i></a>
+              </li>
+              <li>
+                <a href="#"><i class="fa-brands fa-twitter"></i></a>
+              </li>
+              <li><a href="#">English</a></li>
+              <li><a href="#">My Page</a></li>
+              <li><a href="#">Logout</a></li>
+            </ul>
+          </nav>
+          <nav class="nav-bar-two">
+            <div class="">
+              <ul>
+                <li><a href="index.html">Gaming convention 2022</a></li>
+              </ul>
+            </div>
+            <div class="navs-two">
+              <ul>
+                <li><a href="about.html">About</a></li>
+                <li><a href="#program">Program</a></li>
+                <li><a href="">Join</a></li>
+                <li><a href="">Sponsor</a></li>
+                <li><a href="">News</a></li>
+              </ul>
+            </div>
+            <div class="campaign">
```

Comment:
- [ ] Kindly reduce the size of this campaign button so it looks more like the provided design

Your design
![image](https://user-images.githubusercontent.com/81932478/169435019-f012e5a7-eaca-4d24-a2ae-3175f4e4a93c.png)

Provided design
![image](https://user-images.githubusercontent.com/81932478/169435070-f3516862-bc21-468b-ad96-0517edca0d41.png)

Reply:
fixed

Reply:
Implemented 👍